README.md

```diff
@@ -48,18 +54,17 @@ then in your `glance.yml` config file, add the following:
 | `same-tab`      | Open the URL in the same tab. Value customizable per container                                                           | `false`             |
 | `ignore-status` | Status of the containers will not be displayed                                                                           | `false`             |
 
-## Configuration
+## Configuration (required)
 
 Then, for every container you want to monitor, add the following labels to its compose file:
 
 ```yaml
 labels:
   glance.0.enable: true
-  glance.0.name: Sonarr
-  glance.0.description: TV show search
-  glance.0.group: media
-  glance.0.url: http://sonarr.lan
-  glance.0.icon: ./assets/imgs/television-classic.svg
+  glance.0.name: Glance
+  glance.0.description: A self-hosted dashboard that puts all your feeds in one place
+  glance.0.url: https://glance.example.com
+  glance.0.icon: https://avatars.githubusercontent.com/u/159397742?s=48&v=4
```

Comment:
I think it's not the right example to add.
Nobody will have a widget pointing to glance itself; also this may result confusing for users; they may ask theirself: should I put this on glance container?

I put Sonarr as example to say "your container XYZ". A more generic widget should be provided as example.
What a good example could be? I declare all available properties below, so an example like this
```yaml
glance.X.name: <name of the container>
```
isn't quite right

Reply:
Yea, I can see how this might be more confusing. Thanks for explaining your thinking behind the Sonarr example.